_posts/2020-05-05-archivesspace-cleanup-agents.md

```diff
@@ -0,0 +1,71 @@
+---
+title: ArchivesSpace Cleanup: Wrangling Agents in the Wild
+date: 2020-05-05T11:10:00
+author:
+- Amy Berish
+- Katie Martin
+- Darren Young
+layout: post
+categories:
+  - Collaboration
+  - Data Cleanup
+tags:
+  - ArchivesSpace
+  - data
+  - description
+  - discovery
+  - learning
+excerpt_separator: <!--more-->
+---
+
+In preparation for our new discovery system, processing archivists at the RAC have been working on [cleaning up some of our data](https://blog.rockarch.org/archivesspace-clean-up-an-outline) in ArchivesSpace in order to facilitate improved searching. A large chunk of data that has been problematic for some time is our collection of agent records in ArchivesSpace. We simply had too many agents, none of which are standardized, and most which were duplicates of agents that were already in our system, misspelled, or contained incorrect data. Some of our finding aids had too many agents at the file-level, and other findings aids didn’t have any linked agents at all. The state of our agent records makes it difficult for researchers to use the “Filter by Creator” feature, since there is no consistency as to how many agents are assigned, or the level in which they are assigned to a finding aid. In this post, we’ll discuss how we evaluated and cleaned up the agents in our repository.
+<!--more-->
+
+Of course, the varied and complex issues surrounding our agents prompted us to continuously reinvestigate the problems we were attempting to resolve and alter our strategy. Our initial plan to address the many, many agents in our repository was to write a script to remove the links to all agents that have no “source”. Through thinking about this more, and looking at our agent data, we realized that just because an agent doesn’t have a proper source, like LCNAF or local, doesn’t mean it’s not a valid agent. There were plenty of agent records that were missing a source and proved to be linked appropriately to data in ArchivesSpace. We realized we needed to take a much more in-depth look at our agent data. 
+
+## Mo Agents, Mo Problems
+With assistance from the Digital Strategies Team, we were able to get a .csv file export of all agent data in our ArchivesSpace repository. Having all of our agent data in a spreadsheet gave us the ability to use filters in Excel to sort the data alphabetically, or by its source ID (LCNAF, local, or none). This allowed us to take notice of the sheer amount of agents as well as some of the problematic names and acronyms.
+
+![Merging agents in ArchivesSpace](agentspreadsheet.png)
```

Comment:
The image links should look like this: ({{site.baseurl}}/assets/img/2020/04/dimes-agent.png)

Just replace the filename with your own.